samples/zip-split-rabbit-binder/README.adoc

```diff
@@ -0,0 +1,59 @@
+= Split unzipped content into Rabbit Binder destination
+
+This sample demonstrates how out of the box from catalog and custom functions can be composed, and how the final result can be bound to RabbitMQ destination by the https://spring.io/projects/spring-cloud-function[Spring Cloud Stream] framework.
+The goal of this sample is to poll zip files, unzip them, and emit messages to RabbitMQ for every line of text from those zip entries.
+
+The sample uses these dependencies:
+
+[source,groovy]
+----
+implementation 'org.springframework.cloud:spring-cloud-stream-binder-rabbit'
+implementation 'org.springframework.integration:spring-integration-zip'
+implementation 'org.springframework.cloud.fn:spring-file-supplier'
+implementation 'org.springframework.cloud.fn:spring-splitter-function'
+----
+
+The first one is for binding output of the composed function into a RabbitMQ destination.
+The second one is for `UnZipTransformer`, which we use for a custom function to unzip polled files by the `fileSupplier`.
+The `splitterFunction` is used in a `FileSplitter` mode to read lines from unzipped entries and emit each of them as an individual message.
+Essentially, we are splitting twice: zip entries, and content of each file.
+
+The composition is like this: `fileSupplier|unzipFunction|splitterFunction|flattenFunction`.
+(The `flattenFunction` will be explained latter).
```

Comment:
I would move this down below the next line and maybe add an admonition like:

> NOTE: the use of the `flattenFunction` is explained more below

Reply:
Well, for me parentheses in the text come with the same meaning.
Plus this might go away when we fix `splitterFunction` in the next version.